### DISCUSSION TITLE
Do "wait timer" count towards your billable time

### DISCUSSION THREAD

**Discussion by marcelbirkner** [community/community#147534] [Enterprise] [answered]
### Select Topic Area

Question

### Body

We use wait-timer but I cannot find any information if the wait time counts towards our billable time.

https://docs.github.com/en/enterprise-cloud@latest/actions/managing-workflow-runs-and-deployments/managing-deployments/managing-environments-for-deployment#wait-timer

Example for a wait-timer:
<img width="795" alt="image" src="https://github.com/user-attachments/assets/fb945aac-b2a0-4326-8172-d58d0bc3b1f2" />

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @marcelbirkner 👋🏼 , 

Thank you for your patience. Wait time does not count towards your billable time. I'm going to open up a PR to add this to our documentation - appreciate you helping us improve! 

Happy New Year!
